android/res/xml/locales_config.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<locale-config xmlns:android="http://schemas.android.com/apk/res/android">
+    <locale android:name="en" />
+    <locale android:name="ar" />
+    <locale android:name="be" />
+    <locale android:name="bg" />
+    <locale android:name="ca" />
+    <locale android:name="cs" />
+    <locale android:name="da" />
+    <locale android:name="de" />
+    <locale android:name="el" />
+    <locale android:name="en-rGB" />
```

Comment:
Are you sure about en-rGB? Should it be en-GB, as in the example?

Reply:
Did you test these languages on your device? Did they work?

Reply:
I have test with english
- en_US (default english)
<img src="https://user-images.githubusercontent.com/87148630/195440640-b73e999b-9cf3-4e49-985e-7714a43fc1fc.png" height="500"/>

-en_GB
<img src="https://user-images.githubusercontent.com/87148630/195440840-e60d1b96-f4cc-49f7-a532-25a7b80b6bf1.png" height="500" />

Reply:
What about Chinese?

Reply:
zh-Hans
<img src="https://user-images.githubusercontent.com/87148630/195567788-6a039942-ab40-4cce-8516-395936283998.png" height="500"/>

zh-Hant
<img src="https://user-images.githubusercontent.com/87148630/195567911-36f497fb-1805-4ba7-9e57-2424da542700.png" height="500"/>